lodash/lodash.d.ts

```diff
@@ -6412,6 +6412,7 @@ declare module _ {
         endsWith(str?: string, target?: string, position?: number): boolean;
         escape(str?: string): string;
         escapeRegExp(str?: string): string;
+		isMatch(str?: string, target?: string): boolean;
```

Comment:
I think your definition can't compile `_.isMatch(object, { 'age': 36 });`. isn't it?